pug/list_testTypes.pug

```diff
@@ -1,123 +1,132 @@
-extend default
 
+extend default
 include common.pug
 
+
 block vars
-  - var page_title = "Test Types";
-  
+  - var page_title = "All Test Types";
+
+
 block content
   .container-fluid
     pre #{''}
     pre #{''}
   
     if(permissions.userHas(user, "forms:edit"))
-      a.NewTestType.btn.btn-outline-primary(href = '#', data-toggle = "modal" data-target = "#NewTestTypeModal") Create New Test Type Form
+      a(href = '#', data-toggle = "modal" data-target = "#NewTestTypeModal").NewTestType.btn.btn-warning
+        img.small-icon(src = '/images/edit_icon.svg')
+        | &nbsp; Create New Test Type Form
     
       #NewTestTypeModal.modal.fade(tabindex = '-1' role = 'dialog' aria-labelledby = 'NewTestTypeLabel' aria-hidden = 'true')
         .modal-dialog(role = "document")
           .modal-content
             .modal-header
-              h5#NewTestTypeLabel.modal-title New Test Type
+              h5#NewTestTypeLabel.modal-title Create New Test Type
               button.close(type = "button", data-dismiss = "modal", aria-label = "Close")
                 span(aria-hidden = true) &times;
             .modal-body
               div
                 | Enter the new test type's form ID.
                 br
-                | NOTE: this ID should have only underscores and alphanumerics - the human-readable name will be set on the next page.
-                input.NewTestTypeName.form-control(type = "text")
+                | NOTE: this ID should have only underscores and alphanumerics (no spaces) - the human-readable name will be set on the next page.
+              pre #{''}
+              input.NewTestTypeName.form-control(type = "text")
```

Comment:
Just as an FYI:  we can add a `pattern` attribute to input elements for client-side input validation.  We can also automatically remove unwanted characters, ie:  `input.value.replace(/\s/g, '');` to replace all spaces with empty string.